config.go

```diff
@@ -30,7 +30,7 @@ var tokenLimits = map[string]int{
 
 type Config struct {
 	Backend     string  `yaml:"backend"`
-	Model       string  `yaml:"modelName"`
```

Comment:
@tmc I'm unsure about this one

Reply:
Yeah this should be aligned with the flag name so this is preferred.